### ISSUE TITLE
Bugs to Note

### ISSUE THREAD

**Issue by MediaMeasure** [cdgraff/icecast-logs-parser#3]
I know it must have been many years since you must have thought out this.

I have found your code helpful in allowing me to setup icecast log parsing.

For future stumblers there is two major bugs that needs to be corrected/addressed:

1. If a log file has 199 lines, this code will only capture the first full 100 entries as the insert is only done under the "if hits_counter == HIST_PER_QUERY" section.

2. The "values_to_insert" array needs to be cleared after an insert. if not, you will be compounding duplicate entries.

**Comment by cdgraff** [comment]
Yes old but useful as you say. PR are welcome. I will be happy to merge.

**Comment by kpfa** [comment]
@MediaMeasure any chance you could submit a PR or demonstrate a fix for number 2?

**Comment by cdgraff** [comment]
thanks @kpfa merged your changes!
